src/main/java/frc/robot/Constants.java

```diff
@@ -81,4 +81,3 @@ public final class Constants {
     public static String redArmAxis = "Y";
     public static String yellowArmAxis = "X";
     public static String tiltArmAxis = "Z";
-}
```

Comment:
Deleting this curly bracket is causing in the code to fail compilation. You need to put this back in. Other than that, your changes look good.